R/ast-classes.R

```diff
@@ -1489,8 +1500,8 @@ new_node_by_chr <- function(head_chr, is_call = FALSE) {
   ast_node
 }
 
-random_var_name <- function() {
-  paste0("__armacmp_", substr(digest::sha1(stats::runif(1)), 1, 6))
+make_random_var_name <- function(id) {
+  paste0("__arma_tmp_", id)
```

Comment:
I seem to remember the `__` prefix being reserved for identifiers. How about `armcmp_tmp_`?

Reply:
Ah ok. Did not now. That's what I had first. Thanks!